pkg/comp-functions/functions/common/instance_namespace.go

```diff
@@ -48,6 +48,17 @@ func BootstrapInstanceNs(ctx context.Context, comp Composite, serviceName, names
 		return fmt.Errorf("cannot create rbac rules for %s instance: %w", serviceName, err)
 	}
 
+	l.Info("Creating namespace policy to allow access to " + serviceName + " instance")
+	sourceNS := append(comp.GetAllowedNamespaces(), comp.GetClaimNamespace())
+	// Was only available for MariaDB though if it's not enabled, no new namespaces will be added
+	if svc.GetBoolFromCompositionConfig("slosEnabled") {
```

Comment:
Not sure if this flag is necessary, I haven't seen it anywhere in the config. Meanwhile inside `CreateNetworkPolicy() `function there is the `sliNS` being checked, maybe it's the same flag repurposed?

Reply:
After another round of debugging it seems we do not need `slosEnabled `. It's not even used in mariadb, it's always false

Reply:
Grepping over the component, it doesn't look like it's used anywhere. Maybe a leftover?

I'd suggest to get rid of this if.